deployment/terraform/assets/dashboard_data.json

```diff
@@ -21,6 +21,7 @@
   "graphTooltip": 1,
   "id": 3,
   "links": [],
+  "liveNow": false,
```

Comment:
Is this disabling auto refresh?

Reply:
Not sure. Shouldn't matter much since it's internal anyways.